scripts/scripts.js

```diff
@@ -57,11 +60,25 @@ const miloLibs = setLibs(LIBS);
   });
 }());
 
+const setUserProfile = async () => {
+  let profileData = getLocalStorage('profile');
+  if (profileData === null) {
+    const response = await fetch(PROFILE_URL, {
```

Comment:
You may want to use a `try` and `catch` block to handle errors being thrown when either fetching the raw data from API or awaiting JSON data from raw data